account_cryptocurrency/README.rst

```diff
@@ -0,0 +1,151 @@
+======================
+Account Cryptocurrency
+======================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fcurrency-lightgray.png?logo=github
+    :target: https://github.com/OCA/currency/tree/14.0/account_cryptocurrency
+    :alt: OCA/currency
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/currency-14-0/currency-14-0-account_cryptocurrency
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/259/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module provides basic support for the management of cryptocurrencies.
+
+#. Cryptocurrencies are commonly considered property from a
+   taxation standpoint.
+
+#. This means that every time that you receive or send cryptocurrencies, you
+   are entering into a taxable transaction, and will need to report on the
+   gain/loss resulting from the receipt/issue of the cryptocurrency.
+
+#. When you receive cryptocurrencies (in exchange for goods delivered,
+   service rendered or because you mined it), that translates into taxable
+   income to the recipient. The valuation should be based on the market value of
+   the cryptocurrency at the time of receipt, and it can be provided by a
+   third party (e.g. the exchange platform) or needs to be identified by
+   you somehow.
+
+#. You have to retain the cost of the crytptocurrency for as long as you keep
+   it.
+
+#. When you pay out cryptocurrencies (in exchange for goods or services
+   purchased, or if you exchanged for another crypto or normal currency) this
+   constitutes again a taxable transaction. In that case the gain/loss is
+   computed as the difference between the cost of the cryptocurrency that
+   you pay with, and the value of what you receive back, valued in your
+   operating normal currency.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+Configuration
+=============
```

Comment:
```suggestion
```
In the readme/CONFIGURE.rst file, delete this

Reply:
Didn't notice, thanks!